libraries/I2S/src/I2S.h

```diff
@@ -21,7 +21,11 @@
 
 #pragma once
 #include <Arduino.h>
+#include <hardware/pio.h>
+#include <pico/stdlib.h>
```

Comment:
Why are we pulling in the SDK's `pico/stdlib`?  Is this just left over from some debugging?  Can it be removed?